packages/bindings-test/src/chain_communication.rs

```diff
@@ -163,7 +166,7 @@ impl DesmosCli {
     pub fn execute_contract(
         &self,
         contract: &str,
-        msgs: impl IntoIterator<Item = impl Into<CosmosMsg>>,
+        msgs: impl IntoIterator<Item = CosmosMsg>,
```

Comment:
Fix it since `impl Into<CosmosMsg>` will allow only one type.

For instance, the following array would be error:
```rust
[
  request_transfer_msg,
  cancel_transfer_msg,
]
```